cmake/windows/resources/installer-Windows.iss.in

```diff
@@ -1,4 +1,4 @@
-#define MyAppName "@CMAKE_PROJECT_NAME@"
+#define MyAppName "@PLUGIN_DISPLAY_NAME@"
```

Comment:
Don't do this in this PR that never touches this file.
Do this in https://github.com/DistroAV/DistroAV/pull/1134 that already touches this file.

Reply:
my bad had the wrong branch open -_-. Fixed.